src/handlers/motions/motionFinalized/motionFinalized.ts

```diff
@@ -30,6 +34,20 @@ export default async (event: ContractEvent): Promise<void> => {
       motionData,
     } = finalizedMotion;
 
+    const {
+      revealedVotes: {
+        raw: { yay: yayVotes, nay: nayVotes },
+      },
+      motionStakes: {
+        percentage: { yay: yayPercentage, nay: nayPercentage },
+      },
+    } = motionData;
```

Comment:
We're already destructuring motionData above (usersStakes) so these can be combined